website/README.md

```diff
@@ -0,0 +1,197 @@
+This website was created with [Docusaurus](https://docusaurus.io/).
```

Comment:
Is this file generated? If it is, please add to the gitignore, if not instead of pasting this please just provide a link to docusaurus documentation.

Reply:
not generated.  This is the package that does all the work and it has a package.json file